### PR TITLE
working with flat coordinates

### PR DESCRIPTION
**Pull request by Rafael-G-C**
I changed the files to give and use flat coordinates (array of 3*N size)

### CODE REVIEW

File: dftd3/dftd3.py [hunk 1]
```diff
@@ -87,19 +87,11 @@ def d3(
     attractive_r6_vdw = 0.0
     # van der Waals attractive R^-8
     attractive_r8_vdw = 0.0
-    # Axilrod-Teller-Muto 3-body repulsive
+    # Axilrod-Teller-Mu1to 3-body repulsive
```
Comment (robertodr) [inline dftd3/dftd3.py]:
```suggestion
    # Axilrod-Teller-Muto 3-body repulsive
```

File: dftd3/utils.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def getMollist(bondmatrix, startatom):
     return atomlist
 
 
-def ncoord(natom, atomtype, xco, yco, zco, k1=16, k2=4 / 3):
+def ncoord(natom, atomtype, flat_coords, k1=16, k2=4 / 3):
```
Comment (robertodr) [inline dftd3/utils.py]:
```suggestion
def ncoord(atomtype, coordinates, k1=16, k2=4 / 3):
```

File: dftd3/utils.py [hunk 3]
```diff
@@ -77,14 +77,15 @@ def ncoord(natom, atomtype, xco, yco, zco, k1=16, k2=4 / 3):
     """
 
     cn = []
```
Comment (robertodr) [inline dftd3/utils.py:80]:
```suggestion
    cn = []
    natom = len(atomtypes)
    
    if natom != len(coordinates) // 3:
         raise RuntimeError("The size of the coordinates and atom types arrays do not match") 
```

**Review by robertodr — COMMENTED** [review]
Good job. Few small changes to add.

**Comment by Rafael-G-C** [comment]
did not see those, let me commit them right now

**Review by robertodr — APPROVED** [review]

**Pull request merged**